### DISCUSSION TITLE
Can I disable the 'nspanel' wifi network

### DISCUSSION THREAD

**Discussion by hapklaar** [Blackymas/NSPanel_HA_Blueprint#1871] [Q&A] [answered]
Today I noticed I have a 'nspanel' network/ssid in my wifi list. Why is it there and can it be disabled?

**Comment by edwardtfn** [comment, marked as answer]
This is the fallback network, which is enabled when your panel fails to connect to the Wi-Fi network and can be useful to troubleshoot your panel when that happens.

You should look on why your panel is losing the Wi-Fi connection, as solving that will prevent your panel exposing the fallback access point.

If you want to disable the fallback AP anyways, please try adding this customization to your panel's yaml:

```yaml
wifi:
  ap: !remove
```

**Reply by hapklaar** [reply]
Ah I see. At the time when I saw the network, I had no wifi issues but had rebooted the AP some time before. Guess it takes a short while to disappear after the wifi network is back up. It's not in the list anymore now.

thanks for answering!
